### PR TITLE
[ADD] scheduler_error_mailer_therp

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: scheduler_error_mailer_therp/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "depends": [
+        'scheduler_error_mailer',
+    ],
+    "data": [
+        "data/email_template.xml",
+    ],
+    "installable": True,
+    "application": False,
+    "post_init_hook": 'post_init_hook',
+    "external_dependencies": {
```
Comment (NL66278) [inline scheduler_error_mailer_therp/__openerp__.py]: Does the external_dependencies key make sense here?

Comment (hbrunn) [inline scheduler_error_mailer_therp/__openerp__.py]: @NL66278 thanks, not at all. It's just a (bad?) habit of mine to leave stuff from the module template that doesn't hurt

**Comment by NL66278** [comment]
:+1:

**Comment by hbrunn** [comment]
@Therp/core please review/merge

File: scheduler_error_mailer_therp/README.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+==========================================
+Configure scheduler_error_mailer for Therp
+==========================================
+
+This module configures cronjobs in a way that exceptions raised in cron jobs will be mailed to Therp's helpdesk.
+
+If you're not affliated with Therp, this module won't do you any good, but you're welcome to copy it in order to have your own version of it. Just be sure to change the recipient email address of the mail template then.
```
Comment (NL66278) [inline scheduler_error_mailer_therp/README.rst]: Spelling error in affiliated

**Comment by NL66278** [comment]
@hbrunn Except for the spelling error ready to be merged.

**Comment by hbrunn** [comment]
done & rebased

**Pull request merged**